dashboard/routes/wallet/open.go

```diff
@@ -40,6 +40,13 @@ func WalletOpenRouter(router *gin.RouterGroup) {
 				context.AbortWithError(http.StatusForbidden, err)
 				return
 			}
+
+			err = password.SavePassword()
```

Comment:
Why do we need to save password when opening wallet here?

Reply:
if wallet file exists, --password-file parameters can work?

Reply:
I think it's better to save password to file only when creating wallet but not opening wallet, otherwise if user used dashboard once, the password plaintext will be saved to disk...